cmd/gateway-gcs.go

```diff
@@ -39,11 +38,36 @@ import (
 )
 
 const (
-	// ZZZZMinioPrefix is used for metadata and multiparts. The prefix is being filtered out,
-	// hence the naming of ZZZZ (last prefix)
-	ZZZZMinioPrefix = "ZZZZ-Minio"
+	// gcsMinioMeta is used for multiparts. We have "minio.sys.temp" prefix so that
+	// listing on the GCS lists this entry in the end. Also in the gateway
+	// ListObjects we filter out this entry.
+	gcsMinioPath = "minio.sys.temp"
```

Comment:
In my testing, I found that - ListObjects does list the minio.sys.temp entry (you can try with `mc`). This is perhaps a bug? Or the comment above needs to be updated.